pkg/proxy/request.go

```diff
@@ -27,6 +28,7 @@ const (
 func httpValidator(r *http.Request) url.Values {
 	rules := govalidator.MapData{
 		"type":            []string{"required"},
+		"overwrite":       []string{"in:1,True,true,t,T,TRUE,0,False,false,f,F,FALSE"},
```

Comment:
if it is boolean and according to https://swagger.io/docs/specification/data-models/data-types/ should only be `true` - `false`.

Reply:
These are accepted values for validator because go parseBool accepts these.

https://golang.org/pkg/strconv/#ParseBool

But if you don't like, I can change

Reply:
Got it, just want make it consistent to our open API specs

Reply:
So should I change or leave?